lightning/src/blinded_path/payment.rs

```diff
@@ -148,6 +179,34 @@ pub(super) fn blinded_hops<T: secp256k1::Signing + secp256k1::Verification>(
 	Ok(blinded_hops)
 }
 
+pub(super) fn compute_payinfo(
+	path: &[(PublicKey, BlindedPaymentTlvs)]
+) -> Result<BlindedPayInfo, ()> {
+	let mut curr_base_fee: u128 = 0;
+	let mut curr_prop_mil: u128 = 0;
+	for (_, payment_tlvs) in path.iter().rev().skip(1) {
```

Comment:
Should we ensure that the path contains the correctly ordered `BlindedPaymentTlvs` variants?